src/studio/src/designer/frontend/packages/schema-editor/src/utils.ts

```diff
@@ -89,26 +94,29 @@ export function createJsonSchemaItem(uiSchemaItem: UiSchemaItem | any): any {
   return item;
 }
 
-export function buildUISchema(schema: any, rootPath: string, includeDisplayName?: boolean): UiSchemaItem[] {
+export function buildUISchema(schema: any, rootPath: string, includeDisplayName: boolean = true): UiSchemaItem[] {
   const result : UiSchemaItem[] = [];
   if (typeof schema !== 'object') {
+    // What is this...
```

Comment:
It's been a while since I worked on this :) But I think that since `buildUISchema` is recursive, this is to catch the actual value (f.ex. string/number, etc) after the code has iterated through an object and ended up with a key/value pair where the value is not an object that needs to be iterated further.

Reply:
Ah okey, thanks!